src/contracts/pods/EigenPod.sol

```diff
@@ -213,7 +213,7 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
                 oracleTimestamp,
                 validatorIndices[i],
                 stateRootProof.beaconStateRoot,
-                balanceUpdateProofs[i],
```

Comment:
can we add a comment here just explaining why we are using the fields proof here?  Just saying that the validator fields contains the effective balance